### PR TITLE
github: allow release workflow to wait up to 15 minutes for npm package to publish

### LINKED ISSUE #2999: Releases not uploaded to Dockerhub
**Describe the bug**

At least version 0.28.0 is not uploaded to Dockerhub. The publish to Dockerhub action failed. https://github.com/ChainSafe/lodestar/runs/3255490028

```
Step 5/12 : ENV VERSION=$VERSION
 ---> Running in ce678a409691
Removing intermediate container ce678a409691
 ---> 47fbf0c68360
Step 6/12 : RUN npm install @chainsafe/lodestar-cli@$VERSION
 ---> Running in eade1e96c7d9
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @chainsafe/lodestar-cli@0.28.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-08-05T18_50_04_526Z-debug.log
The command '/bin/sh -c npm install @chainsafe/lodestar-cli@$VERSION' returned a non-zero code: 1
```

I think the problem is a race condition with NPM. We have a prior action that uploads that version to NPM, then run the Dockerfile that installs from NPM. Since the timing is tight, it may happen that NPM doesn't acknowledge the existence of that version.

**Expected behavior**

All releases should be uploaded to Dockerhub

### PR DESCRIPTION
**Pull request by q9f**
**Motivation**

Trying to fix #2999 where releases are no longer pushed to docker hub.

**Description**

The issue was introduced in #2672 unknowingly that dist releases might be delayed in the publication on the npm package registry side.

This adds a script that only runs on release build to verify that the npm version exists before attempting and gracefully takes a nap in case it cannot find the required version.

For now I limited it to ~~60~~ 900 seconds but we can play around with this number. ~~Honestly, 60 should be fine, we can test that though.~~

Remaining question: Why are nightly builds not failing? I was staring at the workflows and nightly releases should face the same issue but they don't. Maybe NPM handles them differently.

Fix #2999

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -116,6 +116,7 @@ jobs:
     if: needs.tag.outputs.tag != ''
     steps:
       - uses: actions/checkout@v2
+      - run: scripts/await-release.sh ${{ needs.tag.outputs.tag }} 60
```
Comment (dapplion) [inline .github/workflows/release.yml]: I would do much more than 60 seconds, like 10 or 20 minutes haha

**Review by dapplion — COMMENTED** [review]
Nice job! :+1:

**Comment by q9f** [comment]
Bumped it to 15 minutes. We can test this with #3012 WDYT?

**Comment by dapplion** [comment]
> Bumped it to 15 minutes. We can test this with #3012 WDYT?

Yeah let's do it

**Review by dapplion — APPROVED** [review]

**Pull request merged**